### PR TITLE
docs: change edge harbor hardware requirements

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR changes the hardware requirements for Edge cluster if they use the Harbor pack. 

## Changed Pages

💻 [Add Preview URL for Page]()

## Jira Tickets

🎫 [PE-4015](https://spectrocloud.atlassian.net/browse/PE-4015)

## Backports

Can this PR be backported?

- [ ] No. This is a release PR. 

[PE-4015]: https://spectrocloud.atlassian.net/browse/PE-4015?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md [hunk 1]
```diff
@@ -42,7 +42,10 @@ documentation.
 
 - Each of your Edge hosts must have at least 4 CPUs and 8 GB of RAM.
 
-- At least 160 GB of persistent storage. The actual amount of storage required depends on the size of your images.
+  - For single-node clusters with Harbor, the Edge host needs at least 6 CPUs and 12 GB of RAM.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md:45]: Lenny is this one worker node one control plane node setup? Or is this something different?

File: docs/docs-content/integrations/harbor-edge.md [hunk 2]
```diff
@@ -37,11 +37,16 @@ cluster. You cannot manually choose a version of this pack.
 
 - All Edge devices in your cluster must have at least 4 CPUs and 8 GB of RAM.
 
-- At least 160 GB of persistent storage volume for the cluster. The actual volume required depends on the size of the
+  - For single-node clusters, your Edge host must have at least 6 CPUs and 12 GB of RAM.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/harbor-edge.md:40]: Same question as above

**Review by karl-cardenas-coding — COMMENTED** [review]
Just a minor question that could use some clarification

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md (hunk 1 above)
Comment (lennessyy) [reply docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md]: It's one node that handles both control plane and worker capabilities

File: docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md [hunk 3]
```diff
@@ -42,7 +42,10 @@ documentation.
 
 - Each of your Edge hosts must have at least 4 CPUs and 8 GB of RAM.
 
-- At least 160 GB of persistent storage. The actual amount of storage required depends on the size of your images.
+  - For single-node clusters, where there is only one Edge host handling both control plane and worker capabilities,
```
Comment (caroldelwing) [inline docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/local-registry.md:45]:
```suggestion
  - For single-node clusters, where there is only one Edge host handling both control plane and worker node capabilities,
```
maybe? :)

File: docs/docs-content/integrations/harbor-edge.md [hunk 4]
```diff
@@ -37,11 +37,17 @@ cluster. You cannot manually choose a version of this pack.
 
 - All Edge devices in your cluster must have at least 4 CPUs and 8 GB of RAM.
 
-- At least 160 GB of persistent storage volume for the cluster. The actual volume required depends on the size of the
+  - For single-node clusters, where there is only one Edge host handling both control plane and worker capabilities,
```
Comment (caroldelwing) [inline docs/docs-content/integrations/harbor-edge.md:40]:
```suggestion
  - For single-node clusters, where there is only one Edge host handling both control plane and worker node capabilities,
```
maybe?

**Review by caroldelwing — APPROVED** [review]
Everything looks good :)

**Pull request merged**